behave_django/environment.py

```diff
@@ -104,10 +106,13 @@ def teardown_test(self, context):
         del context.test
 
 
-def monkey_patch_behave(django_test_runner):
+def monkey_patch_behave(django_test_runner, behave_runner):
     """
     Integrate behave_django in behave via before/after scenario hooks
     """
+    # Override the run_behave to inject the runner_class
```

Comment:
Do we need this comment? Is the `partial` not self-explanatory, maybe? Maybe calling it `override_foo` could make this comment obsolete?